src/lib/HederaLib.ts

```diff
@@ -68,4 +68,13 @@ export class HederaWallet {
       return { error: (e as Error).message }
     }
   }
+
+  public signMessage(bytes: Uint8Array) {
+    const signedMessage = this.privateKey.sign(bytes)
```

Comment:
Nice find on this `privateKey.sign` method. I just updated the class method to return a structured payload

Reply:
perfect